Portfolio: mobile version (1st part)

fbclh opened the pull request:
* Creates the skeleton, HTML and CSS structure for the whole project.
* Creates all the necessary folders, files, assets and linters required for the project.

DammyShittu commented on index.css:
```diff
@@ -0,0 +1,95 @@
+  padding-bottom: 6rem;
+}
+
+.header {
+  margin: 0;
+  padding: 1.5rem;
+  padding-bottom: 0;
+  font-size: 2.5rem;
+  line-height: 3.25rem;
+}
```
- [ ] Kindly follow the color as seen in the design template that you're using. I'm sure you'll love it 😉 
For ease, the color is `#172B4D`

DammyShittu commented on index.css:
```diff
@@ -0,0 +1,95 @@
+  line-height: 3.25rem;
+}
+
+.about p {
```
- [ ] Kindly follow the color as seen in the design template that you're using. I'm sure you'll love it 😉
For ease, the color is `#344563`;

DammyShittu commented on index.css:
```diff
@@ -0,0 +1,95 @@
+  padding-bottom: 0;
+}
+
+.socials {
```
- [ ] To make the social media icons look more like the ones in the template, I would recommend you follow the `background-color` given in the design template 😁 

- [ ] I noticed that the links do not have any interaction styling(Active, Hover and Pressed) set up yet. What do you think about writing these styles? I can assure you it'll really improve user experience 💯.
Please, check the design template to see the appropriate color for each interaction.

DammyShittu reviewed (changes requested):
Hi @fbclh, 

I must say, you have done a really amazing job so far. Well done 👍

- Linters are passing ✔️ 
- Really nice README file ✔️
- Great adherence to design ✔️
- Really well structured code ✔️ 

However, there are some issues that you still need to work on to go to the next project, but you are **ALMOST THERE**!

# REQUIRED CHANGES ♻️

- What do you think about adding the `Getting Started` section to your README file? 🤔  This will serve as a guide to any visitor who would like to check the work you've done on their local machine. This is an [example](https://i.imgur.com/dw4qEzP.png) of the `Getting Started` section.

_Check the other comments under the review._

# Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fbclh commented:
Hello there thanks for your feedback, I have made all the changes below based on your feedback!

* Added Getting Stated to README
* Changed the color of h1, p and icons to match design specs
* As per instructions on the design only buttons not the icons has interaction state style, so I am leaving like that for the time being.

andersonlebon reviewed (approved):
Hi @fbclh  
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: :champagne: 

![nodding-yes](https://user-images.githubusercontent.com/65068771/143095390-344d1cb0-b853-4b91-8662-5d9049dc9e60.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
